bin/authors_update.py

```diff
@@ -71,25 +75,29 @@ def move(l, i1, i2):
 
 git_people.pop(226) # Sergey B Kirpichev
 
-header = """\
-All people who contributed to SymPy by sending at least a patch or more (in the
-order of the date of their first contribution), except those who explicitly
-didn't want to be mentioned. People with a * next to their names are not found
-in the metadata of the git history. This file is generated automatically by
-running `./bin/authors_update.py`.
-"""
-header_extra = """There are a total of {authors_count} authors.\n""".format(authors_count=len(git_people))
-
-with open(os.path.realpath(os.path.join(__file__, os.path.pardir,
-                                        os.path.pardir, "AUTHORS")), "w") as fd:
+header = filldedent("""
```

Comment:
This adds a blank line to the top of the file.

Reply:
fixed -- I use ltrim() on the string now.